notify/notify.go

```diff
@@ -543,9 +543,9 @@ func (r RetryStage) Exec(ctx context.Context, alerts ...*types.Alert) (context.C
 		case <-tick.C:
 			if retry, err := r.integration.Notify(ctx, alerts...); err != nil {
 				numFailedNotifications.WithLabelValues(r.integration.name).Inc()
-				log.Debugf("Notify attempt %d failed: %s", i, err)
+				log.Debugf("Notify attempt %d for %q failed: %s", i, r.integration.name, err)
```

Comment:
Not sure this debug logging actually has any value as we ultimately always [log errors](https://github.com/prometheus/alertmanager/blob/7f1d11132473ace30b23b5cb4d04b742c2149654/dispatch/dispatch.go#L263-L266) happening in the notification pipeline, but I also don't mind keeping it as it's closer to the source here without many moving parts where it may get lost.